### DISCUSSION TITLE
How user can see their own Avatar

### DISCUSSION THREAD

**Discussion by jayvs6341** [mozilla/hubs#5099] [Avatar Questions] [answered]
Hello,

I have setup my Own Hubs cloud on AWS.

Only thing I not get  is,

As a user, when I join the room, how can I see my own avatar.

I can see other users but not own.

**Comment by matthewbcool** [comment, marked as answer]
You can inspect your avatar (desktop press the 'i' key) or use the in-room camera (desktop: 'c' key).

**Comment by jayvs6341** [comment]
Thanks for your reply.

But it seems not a good solution. This is opening camera. If I move from my place, it left behind me.

I want to see my self as a other users, like I am playing game, in which I can see my own character while moving. I can jump , I can walk etc.
